web/pgadmin/browser/static/js/node.js

```diff
@@ -159,7 +159,7 @@ define('pgadmin.browser.node', [
           },
           enable: _.isFunction(self.canDrop) ?
             function() {
-              return !!(self.canDrop(arguments));
+              return !!(self.canDrop.apply(self, arguments));
```

Comment:
we have remove this change to fix sonarqube. please do not add back.
To fix this change it to `self.canDrop(...arguments)`, arguments is array here.
there are 2 more places apart from this, please change there as well.

Reply:
Done.